internal/aws/k8s/go.mod

```diff
@@ -56,6 +57,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/containerinsight => ../../aws/containerinsight
```

Comment:
We can do this but it will need to get undone when you merge.
The alternative would be to just do containerinsight change and then update go.mod to point to your changes (git hash, not a fancy tag)